metals/src/main/scala/scala/meta/internal/metals/WorkspaceLspService.scala

```diff
@@ -930,6 +930,13 @@ class WorkspaceLspService(
           _.cleanCompile(),
           ServerCommands.CleanCompile.title,
         ).asJavaObject
+      case ServerCommands.CompileTarget(target) =>
+        onCurrentFolder(
+          _.compileTarget(new BuildTargetIdentifier(target)),
+          ServerCommands.CompileTarget.title,
+          false,
+          () => false,
+        ).asJavaObject
```

Comment:
```suggestion
        ).liftToLspError.asJavaObject
```
might be needed, since otherwise no error is forwarded. Or do you not needed it here?